### ISSUE TITLE
Superuser's roles are not displayed correctly

### ISSUE THREAD

**Issue by oliverroick** [Cadasta/cadasta-platform#1121]
### Steps to reproduce the error

- In the dev VM navigate to http://localhost:8000/organizations/cadasta/members/iross/

### Actual behavior

- According to the page, _iross_ has no permissions on any of the orgs projects despite being a superuser. The permissions itself are set correctly, meaning the user can manage the org and all projects as an admin. 

### Expected behavior

- _iross_ is a platform superuser, the roles for all projects and the organization should be displayed as administrator.

**Comment by niharika1995** [comment]
@oliverroick

This happens because `iross` isn't an administrator of the organization `cadasta`.

```ipy
In []: OrganizationRole.objects.get(user=iross, organization=cadasta)
Out[]: <OrganizationRole id=xnuqbdjpevh57pwms5cmqj47 user=iross organization=cadasta admin=False>
```
Should `<Role: superuser>` within `iross.assigned_policies()` override the `admin` flag of the `OrganizationRole` object that was queried?

**Comment by niharika1995** [comment]
@oliverroick 
The template context for this view contains `is_administrator=True` and `org_admin=False`. Do the two variable signify the same thing? Also, it also contains `current_user=False` even though the the user `iross` is viewing their own profile. Are these bugs or expected behaviour?

**Comment by oliverroick** [comment]
@niharika1995 Sorry for the belated response, I needed to find some time to look into this first. So `iross` is a special case because they are a superuser.

`is_administrator` is a flag that is usually used to work our wether to display buttons for managing the organization (e.g.; edit membership, edit org details). It is added through [`OrgAdminCheckMixin`](https://github.com/Cadasta/cadasta-platform/blob/master/cadasta/organization/views/mixins.py#L233-L267) and it's used across various organization views. For superusers `is_administrator` is always `True`.

`org_admin` is only available in that particular view. It indicates whether the member is actually and organization admin. [See here](https://github.com/Cadasta/cadasta-platform/blob/master/cadasta/organization/views/default.py#L250-L253). For superusers this is `False` because they are not actually a member of the organization, even though they can manage the organization. 

`current_user` indeed indicates whether the user tries to change their own permissions. It should be `False` for superusers even though it's a bit misleading. Superusers are the only people that can change their own permissions, they are like god, they can do anything. 

So to fix this bug, you would need to display the text "Administrator" for superusers, like it is done on the project member page. We have done something similar to the [`ProjectRoleForm`](https://github.com/Cadasta/cadasta-platform/blob/master/cadasta/organization/views/default.py#L250-L253). I thing solving this for the organization forms is going to be a bit more difficult. But feel free to give it a try.

**Comment by clash99** [comment]
I believe this will be fixed with PR #1651. Do you agree @oliverroick?

**Comment by oliverroick** [comment]
No, that PR doesn't fix it.
